cogs/helpers.py

```diff
@@ -53,11 +56,57 @@
 CURRENCY_TEMPLATE = "http://www.xe.com/currencyconverter/convert/" \
                     "?Amount={}&From={}&To={}"
 
+try:
+    os.mkdir('./pickles')
+except Exception:
+    pass
+
 
 class Helpers():
     def __init__(self, bot):
         self.bot = bot
 
+    async def cfia_rss(self):
```

Comment:
Can you explain this functionality to me? We're caching the last read id into a pickle, this co-routine periodically polls the XML for the latest recall id, and if that is not the cached ID, we send out a recall warning - correct? 

What happens in the case that we get 2 or more new recalls into the feed before refreshing? Isn't it better to increase the refresh timer and parse all missed recalls? - We reduce the risk of missing recalls, and its not critical for us to have recall warnings being posted with minute sensitivity, no? Overall, I think it also reduces load. 

If spamming multiple messages at a time is a concern, we can add a sleep timer to space them out too.

Reply:
I'll change the functionality to generate a dictionary of all current recall notices that aren't in the pickle if it exists. Then, each new notice would be posted to the server and the new recalls would be added to the dictionary, which would be pickled. This whole routine could be run daily.